lib/safe-pg-migrations/plugins/blocking_activity_logger.rb

```diff
@@ -9,13 +9,15 @@ module BlockingActivityLogger
     include Helpers::BlockingActivitySelector
 
     %i[
+      add_check_constraint
       add_column
-      remove_column
       add_foreign_key
-      remove_foreign_key
       change_column_default
       change_column_null
       create_table
+      remove_column
+      remove_foreign_key
+      drop_table
     ].each do |method|
```

Comment:
It's maybe a good idea to declare this list as a constant like for the retrier Class, don't you agree ?